docs/tutorial.md

```diff
@@ -10,8 +10,8 @@
 6. [Repeated](#repeated)
 7. [Map](#map)
 8. [Lens Laws](#lens-laws)
-9. [Example: Person](./person/README.md)
-10. [Example: Coffee Order](./coffee-order/README.md)
+9. [Example: Person](https://github.com/google/proto-lens/tree/master/proto-lens-tutorial/person)
```

Comment:
I tested it out; it doesn't work because the file isn't under the "docs" folder.

Also, I feel it's helpful to link to the concrete code example (especially now that they're the same).